WordPress/Classes/ViewRelated/Site Creation/Verticals/VerticalsCell.swift

```diff
@@ -1,24 +1,32 @@
 import UIKit
+import Gridicons
 
-class VerticalsCell: UITableViewCell, ModelSettableCell {
+final class VerticalsCell: UITableViewCell, SiteVerticalPresenter {
     @IBOutlet weak var title: UILabel!
 
-    var model: SiteVertical? {
+    var vertical: SiteVertical? {
         didSet {
-            title.text = model?.title
+            title.text = vertical?.title
         }
     }
 
     override func awakeFromNib() {
         super.awakeFromNib()
         styleTitle()
+        styleAccessoryView()
     }
 
     override func prepareForReuse() {
         title.text = ""
     }
 
     private func styleTitle() {
+        title.font = WPStyleGuide.fontForTextStyle(.body, fontWeight: .regular)
+        title.textColor = WPStyleGuide.darkGrey()
+    }
 
+    private func styleAccessoryView() {
+        let accessoryImage = Gridicon.iconOfType(.chevronRight).imageWithTintColor(WPStyleGuide.greyLighten20())
+        accessoryView = UIImageView(image: accessoryImage)
```

Comment:
I'm guessing we cannot tint the disclosure indicator without switching to a custom icon? Let's revert to the system icon here and on the segments screen if we also used the same method there as well.

Reply:
Same thing with the clear icon also. If we definitely cant tint it. It's still best for us to use the system icon.